addons/im_livechat/tests/chatbot_common.py

```diff
@@ -1,10 +1,9 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo.tests import common
-from odoo.addons.mail.tests.common import MailCommon
 
 
-class ChatbotCase(MailCommon, common.HttpCase):
```

Comment:
Note: removing `MailCommon` from `ChatbotCase` to be able to combine the latter with `TestLivechatCommon` which also has `MailCommon`.